app/constants/snack_bar.ts

```diff
@@ -27,7 +28,7 @@ export const MESSAGE_TYPE = {
     DEFAULT: 'default',
 };
 
-type SnackBarConfig = {
+export type SnackBarConfig = {
```

Comment:
See comment abouut creating type MessageType vs exporting an internal type.

Reply:
Type coupling has some objective benefits. If we have an established convention or practice of not using it in mobile, I'll revert this change. Because we use it a lot in webapp and there some benefits, I used it here. Please LMK if this needs to be reverted.